packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -45,6 +45,21 @@ static inline CGFloat GetMinDistanceBetweenRectVerticesAndPoint(NSRect rect, NSP
 	);
 }
 
+static NSView *GetFirstKeyViewWithin(NSView *parentView)
+{
+	for (NSView *view in [parentView subviews]) {
+		if ([view canBecomeKeyView]) {
+			return view;
+		}
+
+		NSView *match = GetFirstKeyViewWithin(view);
+		if (match) {
+			return match;
+		}
+	}
+	return nil;
+}
```

Comment:
Comment from @HeyImChris last time: (https://github.com/microsoft/fluentui-react-native/pull/1400#discussion_r799040956): 
> Would it make sense to have a GetLastKeyViewWithin to get the last menu button for when a user launches a menu and presses ArrowUp?

Yes, though I'd rather implement that as a separate change..